packages/shared/helpers/src/rpc-helpers.ts

```diff
@@ -47,4 +47,43 @@ export function getRpcUrl(networkId: number) {
   }
 }
 
+export function getContractUrls(tokenSymbol: string) {
```

Comment:
You should not be using hardcoded URLs. Contract addresses are availble on the TokenDefinition object already.

Reply:
For the different pages it will be the following value from the state:
Variable Lend / Liquidity / Leveraged Liquidity: Collateral
Variable Debt: Debt
Fixed Lend / Fixed Borrow / Leveraged Liquidity: Notional Address via the constants file